backend/tempStorage.js

```diff
@@ -0,0 +1,37 @@
+const tempStorage = {};
```

Comment:
If you use a variable, it will just be active when you're running the program. For example, if you are running the program and the `tempStorage` stores A, B, C, when you shut down the program, the variable becomes empty.
You don't want this. You want when you turn on the program back, A, B, C are still here.
I suggest creating a file named `tempInMemoryStorage.json`. JSON is just a nested hash right? So, create a JSON and make our functions interact with it. It should have a fixed size (let's assume 1000 by default). If the length of JSON file is 1001, we remove the **Last Added** data or **Least Recently Used** data (it's up to you to decide the policy; inform me later).